docs/standard/analyzers/api-analyzer.md

```diff
@@ -0,0 +1,62 @@
+---
+title: .NET API Analyzer
+description: Learn how the .NET API Analyzer can help detecting deprecated APIs and platform compatibility issues.
+author: oliag
+ms.author: mairaw
+ms.date: 12/11/2017
+ms.topic: article
+ms.prod: .net
+ms.technology: dotnet-standard
+---
+
+# .NET API Analyzer
+
+The .NET API Analyzer is a Roslyn analyzer that discovers potential compatibility risks for APIs with different platforms and detecting calls to deprecated APIs. It can be useful for any developers of the .NET family at any stages of development.
+
+API Analyzer comes as [NuGet package](https://www.nuget.org/packages/Microsoft.DotNet.Analyzers.Compatibility/) and after referencing it in a project, it automatically starts monitoring the code and squiggles problematic API usage. You can get suggestions on possible fixes by clicking on the light bulb, which also includes the ability to suppress the warnings.
+
+## Discovering deprecated APIs
+When a deprecated API (for example `WebClient`) is used in a code, API Analyzer squiggles it out in green and shows a light bulb on the left
```

Comment:
\`WebClient\` can be replaced with a link (\<xref:System.Net.WebClient>)
again, squiggles as a verb; its usual meaning as a verb is to squirm or to wriggle. Perhaps "highlights it with a green squiggly line..."?